Migration guide for tabs (#1503)
contentful/forma-36: packages/forma-36-website/src/content/migration-v3-to-v4.mdx

✅ Form Components: We changed how most of the Form components work. For more information on how to migrate your Form components, see [the detailed documentation](https://github.com/contentful/forma-36/blob/forma-v4/MIGRATION.md#form-components).

✅ Note: Some properties of the Note components were migrated or changed. For more information on how to migrate your Note components, see [the detailed documentation](https://github.com/contentful/forma-36/blob/forma-v4/MIGRATION.md#how-to-migrate-your-note-components).

✅ Notification: Some properties of the Notification components were migrated or changed. For more information on how to migrate your Notification components, see [the detailed documentation](https://github.com/contentful/forma-36/blob/forma-v4/MIGRATION.md#how-to-migrate-your-notification-components).

✅ Modal: We changed the way you import the Modal component. For more information on how to migrate your Modal components, see [the detailed documentation](https://github.com/contentful/forma-36/blob/forma-v4/MIGRATION.md#how-to-migrate-your-modal-components).

> ✅ Tabs: The Tabs components API has been improved. For more information on how to migrate your Tabs components, see [the detailed documentation](https://github.com/contentful/forma-36/blob/forma-v4/MIGRATION.md#how-to-migrate-your-tabs-components).

Reviewer: Gui updated phrasing a bit https://github.com/contentful/forma-36/pull/1510/files
```suggestion
✅ Tabs: The Tabs components API has been improved. For more information, see [the detailed documentation](https://github.com/contentful/forma-36/blob/forma-v4/MIGRATION.md#how-to-migrate-your-tabs-components).
```

✅ Grid: Some properties of the Grid components were migrated or changed. For more information on how to migrate your Grid components, see [the detailed documentation](https://github.com/contentful/forma-36/blob/forma-v4/MIGRATION.md#how-to-migrate-your-grid-components).

> ✅ Tabs: The API of Tabs was changed. For more information on how to migrate your Tabs components, see [the detailed documentation](https://github.com/contentful/forma-36/blob/forma-v4/MIGRATION.md#how-to-migrate-your-tabs-components).

Reviewer 2: ```suggestion
✅ Tabs: The Tabs components API has been improved. For more information on how to migrate your Tabs components, see [the detailed documentation](https://github.com/contentful/forma-36/blob/forma-v4/MIGRATION.md#how-to-migrate-your-tabs-components).
```
